scripts/run_benchmarks.sh

```diff
@@ -121,7 +121,8 @@ for chain in ${chains//,/ }; do
 
       OUTPUT=$(
         $ASTAR_COLLATOR benchmark pallet \
-        --chain=$chain \
+        --runtime="./target/release/wbuild/$chain-runtime/$chain_runtime.compact.compressed.wasm" \
+        --genesis-builder=runtime \
```

Comment:
Is it necessary to use this already?

Because it doesn't work properly for `repeats > 1`.
AFAIK it hasn't been fixed in the latest master yet.

Reply:
I made the change after encountering the following warning:

```bash
2024-09-07 17:49:14 Using the chain spec instead of the runtime to generate the genesis state is deprecated. Please remove the `--chain`/`--dev`/`--local` argument, point `--runtime` to your runtime blob and set `--genesis-builder=runtime`. This warning may become a hard error any time after December 2024.
```

You have more experience with this so I will revert to previous `--chain` flag, I haven't extended testing.

Reply:
That's fine, I made a [PR](https://github.com/AstarNetwork/Astar/pull/1319) for this [issue](https://github.com/AstarNetwork/Astar/issues/1316), but haven't proceeded with it due to the issue when using runtime directly.

Once the upstream issues are resolved, we can get rid of this deprecated usage.